### PR TITLE
feat: autocomplete contacts

### PR DESCRIPTION
**Pull request by y-lohse**

### CODE REVIEW

**Comment by y-lohse** [comment]
[(3) [PHO] Auto-complétion sur les contacts (adresse email et url cozy)](https://trello.com/c/8vETTWTm/400-3-pho-auto-compl%C3%A9tion-sur-les-contacts-adresse-email-et-url-cozy)

File: src/ducks/sharing/autosuggest.styl [hunk 1]
```diff
@@ -0,0 +1,43 @@
+@import 'cozy-ui'
+
+.container
+    position relative
+
+.suggestionsContainer
+    position absolute
+    top 100%
+    margin-top 1px
+    z-index 500
```
Comment (GoOz) [inline src/ducks/sharing/autosuggest.styl]: Why `z-index: 500` ? Why 500 and isn't there a better way using the variables?

File: src/ducks/sharing/autosuggest.styl [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    overflow auto
+    border 1px solid silver
+    border-radius 4px
+    background white
+    display none
+
+.suggestionsContainerOpen
+    display block
+
+.suggestionsList
```
Comment (GoOz) [inline src/ducks/sharing/autosuggest.styl]: There's something I don't get. This class `suggestionList` suggest there's a list `<ul>` with bullets (that you remove with CSS) but I can't see this list anywhere.

File: src/ducks/sharing/autosuggest.styl (hunk 1 above)
Comment (y-lohse) [inline src/ducks/sharing/autosuggest.styl]: Oh shit lol, I accidentally left that in. Yes I'm totally using one of our variables instead.

File: src/ducks/sharing/autosuggest.styl (hunk 2 above)
Comment (y-lohse) [inline src/ducks/sharing/autosuggest.styl]: The list is generated by the [Autosuggest](https://github.com/moroshko/react-autosuggest) component ;)

Comment (GoOz) [inline src/ducks/sharing/autosuggest.styl]: aaah ok then. :)

**Pull request merged**